css/styles.css

```
@@ -242,12 +273,17 @@ main .gallery .thumbnails {
   gap: 31px;
 }
 
-main .gallery .thumbnail {
+main .gallery .thumbnail,
+.thumbnail-select {

```

Se quiser seguir o padrão, poderia fazer algo assim:

```suggestion
.gallery .thumbnail,
.gallery .thumbnail-select {

```